.changeset/clever-bottles-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
+---
+
+converts OptionStatus component to functional component with new tests and stories
```

Comment:
Hmm, well, I definitely want to be providing useful information in the changeset. What about "Refactors OptionStatus component of the Radio widget"? Or would it really be better just to leave it empty since these changes don't have an obvious effect?

Reply:
Sorry, should have been clearer.  The first part is useful... just the part about new tests and stories is not. :) 

So I'd say this is great: "Converts OptionStatus component to a functional component"

Reply:
Oh! Gotcha. Perfect. Will change it to that. :)